e2e/tests/odoo-openmrs-flows.spec.ts

```diff
@@ -1,29 +1,32 @@
 import { test, expect } from '@playwright/test';
-import { HomePage } from '../utils/functions/testBase';
-import { patientName } from '../utils/functions/testBase';
+import { Odoo } from '../utils/functions/odoo';
+import { OpenMRS, patientName } from '../utils/functions/openmrs';
 import { O3_URL, ODOO_URL } from '../utils/configs/globalSetup';
 
-let homePage: HomePage;
+let openmrs: OpenMRS;
+let odoo: Odoo;
 
 test.beforeEach(async ({ page }) => {
-  homePage = new HomePage(page);
+  openmrs = new OpenMRS(page);
+  odoo = new Odoo(page);
+
+  await openmrs.initiateLogin();
```

Comment:
Why `initiateLogin()` why not just `login()`?

Reply:
It makes sense.